README.md

```diff
@@ -266,7 +266,7 @@ By default, profiles are created with autoconnect enabled.
 The `mac` address is optional and restricts the profile to be usable only on
 devices with the given MAC address. `mac` is only allowed for `type`
 `ethernet` or `infiniband` to match a non-virtual device with the
-profile.
+profile. The value of `mac` address is string and should use with quotation mark.
```

Comment:
Please add more context for the reader to understand the problem (that yaml might parse it as a float in sexagesimal notation) and/or link to a good external resource that explains this - there might be something on the Ansible website or the yaml spec.

Reply:
also update `ArgValidatorMac` to reject values of type integer!

Reply:
I guess, that already happens... so just make sure that the error message is sensible.

Reply:
updated